app/templates/event/partials/view_event_details_tab.html

```diff
@@ -1,4 +1,4 @@
-<div class="event_info_row title" style="border-bottom: 4px solid {{ event.color }}">
+<div class="event_info_row title" style="border-bottom: 4px solid '{{ event.color }}'">
```

Comment:
You sure there's a need to?

Reply:
VSC marked this as an error, although the code is still running fine without it.
I reverted it.